src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -89,34 +29,111 @@ const propTypes = {
 
     // Controls the visibility of this component.
     isVisible: PropTypes.bool,
+
+    // The session of the logged in person
+    session: PropTypes.shape({
+        // Email of the logged in person
+        email: PropTypes.string,
+    }),
 };
 
 const defaultProps = {
     reportAction: {},
     isMini: false,
     isVisible: false,
+    session: {},
 };
 
-const ReportActionContextMenu = (props) => {
-    const wrapperStyle = getReportActionContextMenuStyles(props.isMini);
-    return props.isVisible && (
-        <View style={wrapperStyle}>
-            {CONTEXT_ACTIONS.map(contextAction => contextAction.shouldShow && (
-                <ReportActionContextMenuItem
-                    icon={contextAction.icon}
-                    text={contextAction.text}
-                    successIcon={contextAction.successIcon}
-                    successText={contextAction.successText}
-                    isMini={props.isMini}
-                    onPress={() => contextAction.onPress(props.reportAction)}
-                    key={contextAction.text}
-                />
-            ))}
-        </View>
-    );
-};
+function copyToClipboard(reportAction) {
+    // If return value is true, we switch the `text` and `icon` on
```

Comment:
not sure, this was mostly code I copied from the const to the variable. I'll still update has it seems fit, since I agree that it would be better for this to be changed.

Reply:
After looking further, I understand what was meant here, but I think it was a comment from an earlier stage of implementing the `copy to clipboard` I don't think it's applicable here so I'm removing, I feel that the [part of the code](https://github.com/Expensify/Expensify.cash/blob/44ffb0f3334eac72000eb9f20d91d5f64643418a/src/pages/home/report/ReportActionContextMenuItem.js#L37) this is refering to already explains what is happening. Let me know if you think this still needs a comment, I think it's reasonably understandable (even I'm not that familiar with lodash) so I think it's fine whithout the comment.